### PR TITLE
Accordion

### PR DESCRIPTION
**Pull request by howers**

### CODE REVIEW

File: src/organisms/Accordion/AccordionPanel/index.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            {this.props.title}
+          </h1>
+          <span className={styles['toggleSwitch']}>
+            <LinkWrapper type='secondary'>
+              {this.state.open ? this.props.closeText : this.props.openText}
+            </LinkWrapper>
+          </span>
+        </header>
+        <section
+          dangerouslySetInnerHTML={this.createMarkup()}
```
Comment (patrickkim) [inline src/organisms/Accordion/AccordionPanel/index.js]: Why not just add children into section vs setting innerHtml?

File: src/organisms/Accordion/AccordionPanel/__tests__/accordion_panel.spec.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import AccordionPanel from '../';
+
+describe('<AccordionPanel />', () => {
+  // let defaultProps;
+  //
+  // beforeEach(() => {
+  //   defaultProps = {
+  //     // Add default props here
+  //   };
+  // });
```
Comment (drewdrewthis) [inline src/organisms/Accordion/AccordionPanel/__tests__/accordion_panel.spec.js]: These comments should be pulled.

File: src/organisms/Accordion/AccordionPanel/index.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  }
+
+  handleClick() {
+    this.setState({
+      open: !this.state.open
+    });
+  }
+
+  createMarkup() {
+    return { __html: this.props.body };
```
Comment (drewdrewthis) [inline src/organisms/Accordion/AccordionPanel/index.js]: Eventually, we should use the markdown converter here.

File: src/organisms/Accordion/__tests__/accordion.spec.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+import Accordion from 'organisms/Accordion';
+
+describe('<Accordion />', () => {
+  // let defaultProps;
+  //
+  // beforeEach(() => {
+  //   defaultProps = {
+  //     // Add default props here
+  //   };
+  // });
```
Comment (drewdrewthis) [inline src/organisms/Accordion/__tests__/accordion.spec.js]: Should remove these comments.

**Pull request merged**